models/evaluator.py

```diff
@@ -0,0 +1,27 @@
+from nltk.translate.bleu_score import sentence_bleu
+
+
+def evaluation(translations, references):
```

Comment:
why not just use, `gold` and `prediction` here instead of `translations` and `references` variables?

Reply:
`translations` refers to translated sentence and `references` refers to source sentences,can you more explain about `gold` and `prediction`?

Reply:
so based on your message, `prediction` is `translations` and `gold` is `references` (for `gold` naming you can use `truth`) too